src/components/browse/ProductCard.tsx

```diff
@@ -0,0 +1,26 @@
+const ProductCard = ({
+  image,
+  title,
+  price,
+}: {
+  image: string;
+  title: string;
+  price: number;
+}) => {
+  return (
+    <div className="card card-compact bg-base-100 w-96 h-96 shadow-xl">
+      <figure>
+        <img src={image} />
+      </figure>
+      <div className="card-body">
+        <h2 className="card-title">{title}</h2>
+        <div className="card-actions justify-end">
+          <p>R{price}</p>
```

Comment:
Does this need any formatting (e.g. rounded to 2 decimals but also padded with trailing zeros if needs be)?